pybossa/core.py

```diff
@@ -296,11 +296,43 @@ def setup_external_services(app):
     setup_twitter_login(app)
     setup_facebook_login(app)
     setup_google_login(app)
+    setup_amnesy_login(app)
+    setup_discourse_integration(app)
```

Comment:
@georgeslabreche I haven't found a way to put bootstrap code inside the plugin so I have to put it here. If you have idea about that please suggest then the code is cleaner: just inside the plugins folder